File: Armaan4pm/BaseAdapter/app/src/main/java/com/example/baseadapter/SimpleAdapterActivity.java
package com.example.baseadapter;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ListView;

import java.util.ArrayList;
import java.util.HashMap;

public class SimpleAdapterActivity extends AppCompatActivity {
ListView listView;
ArrayList<HashMap<String,String>>  userlist;


    String[] number={"324234","342342","3452342","443534","453453","674564","345354","34543534"};
    int[] imagese= {R.drawable.arebic,R.drawable.naturetwo,R.drawable.download,R.drawable.naturetwo,R.drawable.right,R.drawable.sample,R.drawable.weather,R.drawable.weather};
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_simple_adapter);
        listView = findViewById(R.id.listviewsimple);
        userlist = new ArrayList<>();
          for(int i=0;i<number.length;i++)
          {
              HashMap<String,String> hashMap = new HashMap<>();
              hashMap.put("number",number[i]);
              hashMap.put("image",imagese[i]+"");

              userlist.add(hashMap);
          }

          String[] from = {"number","image"};
          int[] to = {R.id.textviewbase,R.id.imageviewbase};

          CustomSimpleAdapter customSimpleAdapter = new CustomSimpleAdapter(getApplicationContext(),userlist,R.layout.baselayout,from,to);
          listView.setAdapter(customSimpleAdapter);

    }
}

File: Rahoul/TypesofAdapter/app/src/main/java/com/example/typesofadapter/SimpleAdapterActivity.kt
package com.example.typesofadapter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ListView
import android.widget.Toast

class SimpleAdapterActivity : AppCompatActivity() {

    var imaglist: IntArray = intArrayOf(R.drawable.arebic,R.drawable.right,R.drawable.nature,R.drawable.naturetwo,R.drawable.sample);
    var namelist: Array<String> = arrayOf("first","second","Third","Fourt","Fift");
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_simple_adapter)
        val listView = findViewById<ListView>(R.id.simplie_listview);

        val arraylist: ArrayList<HashMap<String,String>> = ArrayList()

        for(  i in imaglist.indices){
            val hashMap:HashMap<String,String>  = HashMap()
            hashMap.put("name",namelist[i])
            hashMap.put("image",imaglist[i].toString())

            arraylist.add(hashMap)
        }

        var from :Array<String> = arrayOf("name","image")
        var to:IntArray = intArrayOf(R.id.array_name,R.id.array_image)
        val customSimpleAdapter = CustomSimpleAdapter(applicationContext,arraylist,R.layout.array_rows,from,to)

        listView.adapter = customSimpleAdapter

        listView.setOnItemClickListener { parent, view, position, id ->

            Toast.makeText(applicationContext,namelist[position].toString(),Toast.LENGTH_SHORT).show()
        }

    }
}